magento/pwa-studio
# [bugfix]: Adds meta descriptions to root pages

## Description

This uses the `Meta` component from `react-head` to provide the description meta tag.

I updated all the queries to return it regardless of whether we're on server or not. I then configured each component to utilize the returned value in the rendered HTML.

If lighthouse still complains with this change that means the sample data/backend data needs to be updated. 

[![Image from Gyazo](https://i.gyazo.com/4fb9c7afc48280f71a2b28a2b0130dc9.png)](https://gyazo.com/4fb9c7afc48280f71a2b28a2b0130dc9)

I'm not sure if JS crawlers wait for/allow this sort of async update of meta description. We could possibly provide a default value?

## Related Issue





Closes [PWA-217](https://jira.corp.magento.com/browse/PWA-217).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to the app, check the HTML for a meta description tag.
2. Navigate to the category and check that it is still there.
3. Do the same for a product.

You may notice that `content` is empty for some pages. If that happens it is because the backend is returning `null`. Check the response for the relevant query to see if that is true.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
